libs/input/src/lib/text-field/text-field.spec.tsx

```diff
@@ -0,0 +1,59 @@
+import { render, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import TextField from '.';
+
+jest.mock('.', () => {
+  return jest.fn((props) => {
+    return (
+      <div data-testid="mocked-text-field">
+        <input
+          data-testid="mocked-input"
+          value={props.value}
+          onChange={(e) => props.onChange && props.onChange(e)}
+          disabled={props.disabled}
+        />
+        <label data-testid="mocked-label">{props.label}</label>
+        <div data-testid="mocked-icon" onClick={props.onIconClick}></div>
+      </div>
+    );
+  });
+});
+
+describe('TextField', () => {
+  test('renders without errors', () => {
+    render(<TextField />);
+    const textFieldElement = document.querySelector(
+      '[data-testid="mocked-text-field"]'
+    );
+
+    expect(textFieldElement).toBeInTheDocument();
+  });
+
+  test('handles input change', () => {
+    const { container } = render(<TextField />);
+    const inputElement = container.querySelector(
```

Comment:
```suggestion
    const inputElement = container.getByTestId('mocked-input')
```
but not sure if this work for mock